components/Addressing/interfaces.rst

```diff
@@ -0,0 +1,99 @@
+Interfaces
+==========
+
+Model Interfaces
+----------------
+
+.. _component_addressing_model_address-interface:
+
+AddressInterface
+~~~~~~~~~~~~~~~~
+
+This interface should be implemented by a model representing the customer's address.
+
+.. note::
+   This interface extends :ref:`component_resource_model_timestampable-interface`. |br|
+   For more detailed information go to `Sylius API AddressInterface`_.
+
+.. _Sylius API AddressInterface: http://api.sylius.org/Sylius/Component/Addressing/Model/AddressInterface.html
+
+.. _component_addressing_model_country-interface:
+
+CountryInterface
+~~~~~~~~~~~~~~~~
+
+This interfaces should be implemented by a model representing a country.
+
+.. note::
+   This interface extends :ref:`component_resource_model_toggleable-interface`. |br|
+   For more detailed information go to `Sylius API CountryInterface`_.
+
+.. _Sylius API CountryInterface: http://api.sylius.org/Sylius/Component/Addressing/Model/CountryInterface.html
+
+.. _component_addressing_model_province-interface:
+
+ProvinceInterface
+~~~~~~~~~~~~~~~~~
+
+This interface should be implemented by a model representing a part of a country.
+
+.. note::
+   For more detailed information go to `Sylius API ProvinceInterface`_.
+
+.. _Sylius API ProvinceInterface: http://api.sylius.org/Sylius/Component/Addressing/Model/ProvinceInterface.html
+
+.. _component_addressing_model_zone-interface:
+
+ZoneInterface
+~~~~~~~~~~~~~
+
+This interface should be implemented by a model representing a single zone. |br|
+It also holds all the :doc:`/components/Addressing/zone_types`.
+
+.. note::
+   For more detailed information go to `Sylius API ZoneInterface`_.
+
+.. _Sylius API ZoneInterface: http://api.sylius.org/Sylius/Component/Addressing/Model/ZoneInterface.html
+
+
+.. _component_addressing_model_zone-member-interface:
+
+ZoneMemberInterface
+~~~~~~~~~~~~~~~~~~~
+
+This interface should be implemented by any model you would like to characterize, as something a specific
```

Comment:
`This interface should be implemented by models that represent something that can be a memeber of a **Zone**, e.g. all countries in the European Union.`